scripts/telemetry.js

```
@@ -1,37 +1,48 @@
 var telemetry = (function() {
 
-    var queue = [];
     var _backendURL = '';
+    var _backendWebSocketURL = '';
     var httpRequest = new XMLHttpRequest();
-
-    var _isQueing = false;
+    var _fallback = false;
 	var _sessionURL;
 	var _isSessionSet = false;
+    var _ws = null;
+    var _captureKeys = false;
 
     var getTimestamp = function() {
         return new Date().toLocaleString();
     };
 
     var _makeRequest = function() {
         if(httpRequest.readyState === XMLHttpRequest.DONE) {
-            if(httpRequest.status === 200) {
-                queue = [];
-            } else {
+            if(httpRequest.status !== 200) {
```

the http errors are not being reported